PC/svm/includes/common.h

```diff
@@ -0,0 +1,73 @@
+#ifndef COMMON
+#define COMMON
```

Comment:
I would keep COMMON_H convention for header guards.

Reply:
Done